[Frontend] Create Website Footer

clairezzhang opened the pull request:
Created a basic footer for the site that sticks to the bottom of the page even when the page is empty. The footer contains the Chigame logo and links to important pages on the site.

<img width="1470" alt="Screenshot 2025-04-27 at 9 25 58 PM" src="https://github.com/user-attachments/assets/145054f5-544a-4032-bc7d-8a13ac116123" />

clairezzhang committed:
[frontend] WIP: Created basic footer

Footer exists and is at bottom of page even when page is empty.

WIP: #548

clairezzhang committed:
[frontend] Finished working footer

Created a working footer based on footer in homepage prototype.

Closes: #548

clairezzhang committed:
Merge branch 'dev' into frontend/create-footer

pjacobsen1 reviewed (approved):
Looks great!

jacqsmei commented on src/templates/base.html:
```diff
@@ -310,5 +317,38 @@
 </script><!-- djlint:on -->
 {# turn djlint back on so that the rest of the code is checked for format/stylistic errors#}
 {% endblock inline_javascript %}
+<footer class="mt-auto">
```
could you fix the indentation in this footer section so that's it's more readable?

clairezzhang committed:
[frontend] Fixed indentation in footer code

Moved footer code to different section and indented it because djLint
was automatically removing all indentation when it was located at the
bottom of the file.

Fixes: #558

jacqsmei reviewed (approved):
changes look good!

clairezzhang committed:
Merge branch 'dev' into frontend/create-footer

clairezzhang committed:
Merge branch 'dev' into frontend/create-footer

clairezzhang committed:
Merge branch 'dev' into frontend/create-footer

clairezzhang committed:
Merge branch 'dev' into frontend/create-footer

jacqsmei commented:
Issue Score: Excellent

Comments:
Nice job! new footer looks great

<!-- {"score": "excellent", "minor": false} -->
